.github/workflows/vmss-auto-shutdown.yaml

```diff
@@ -0,0 +1,59 @@
+name: vmss-auto-shutdown
+on:
+  workflow_dispatch:
+#  schedule:
+#    - cron: "0 20,23 * * *" # Every day at 20:00 and 23:00 BST
+env:
+  DEV_ENV: ${{ secrets.DEV_ENV }}
+permissions:
+  id-token: write
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: "Az CLI login"
+        uses: azure/login@v2
+        with:
+          client-id: 2b6fa9d7-7dba-4600-a58a-5e25554997aa # DTS AKS Auto-Shutdown
+          tenant-id: 531ff96d-0ae9-462a-8d2d-bec7c0b42082 # HMCTS.NET
+          allow-no-subscriptions: true
+
+      - name: Staging - VMSS Auto Stop
+        run: ./scripts/vmss/auto-start-stop.sh deallocate staging
+        env:
+          DEV_ENV: ${{ env.DEV_ENV }}
+
+      - name: Test - VMSS Auto Stop
+        run: ./scripts/vmss/auto-start-stop.sh deallocate test
+        env:
+          DEV_ENV: ${{ env.DEV_ENV }}
+
+      - name: Demo - VMSS Auto Stop
+        run: ./scripts/vmss/auto-start-stop.sh deallocate demo
+        env:
+          DEV_ENV: ${{ env.DEV_ENV }}
+
+      - name: Development - VMSS Auto Stop
+        run: ./scripts/vmss/auto-start-stop.sh deallocate development
+        env:
+          DEV_ENV: ${{ env.DEV_ENV }}
+
+      - name: Sandbox - VMSS Auto Stop
+        run: ./scripts/vmss/auto-start-stop.sh deallocate sandbox
+        env:
+          DEV_ENV: ${{ env.DEV_ENV }}
+
+      - name: ITHC - VMSS Auto Stop
+        run: ./scripts/vmss/auto-start-stop.sh deallocate ithc
+        env:
+          DEV_ENV: ${{ env.DEV_ENV }}
+
+      - name: Untagged - VMSS Auto Stop
+        run: ./scripts/vmss/auto-start-stop.sh deallocate untagged
```

Comment:
Could you check this command to see which Subscriptions are targeted?  it seems to be targeting all the untagged resources